src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,86 @@
-import React from 'react';
+/* eslint-disable padding-line-between-statements */
+/* eslint-disable no-console */
+import React, { useState } from 'react';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { Loader } from '../Loader';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  onAdd: (movie: Movie) => void
+};
+
+export const FindMovie: React.FC<Props> = ({ onAdd }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>();
+  const [movieError, setMovieError] = useState(false);
+  const [isMovie, setIsMovie] = useState(false);
+  const [isLoader, setIsLoader] = useState(false);
+
+  const reset = () => {
+    setIsMovie(false);
+    setQuery('');
+    setMovieError(false);
+    setMovie(null);
+  };
+
+  const handleInputChange = async (
+    event: React.ChangeEvent<HTMLInputElement>,
+  ) => {
+    /* коли я роблю cosnt [value] = event.target, в мене горить лінтер, що тип не підходить, а писати any думаю помага ідея */
+    setQuery(event.target.value);
+
+    if (event.target.value.length === 0) {
+      setMovieError(false);
+    }
+  };
+
+  const handleMovieSubmit = (
+    event: React.ChangeEvent<HTMLFormElement>,
+  ) => {
+    event.preventDefault();
+
+    setIsLoader(true);
+
+    getMovie(query).then((data) => {
+      if ('Error' in data) {
+        setMovieError(true);
+
+        return;
+      }
+
+      // eslint-disable-next-line max-len
+      const img = data.Poster === 'N/A'
+        ? ('https://via.placeholder.com/360x270.png?text=no%20preview')
+        : (data.Poster);
+
+      setMovie({
+        title: data.Title,
+        description: data.Plot,
+        imgUrl: img,
+        imdbUrl: img,
```

Comment:
imdb url should look like this
![Screenshot 2022-07-27 at 18 07 35](https://user-images.githubusercontent.com/99331793/181282742-330e6eda-bb19-45fa-8a3a-ad3114a334e0.png)

Reply:
not the same as imgUrl
![Screenshot 2022-07-27 at 18 20 30](https://user-images.githubusercontent.com/99331793/181285536-e554bc52-37de-4a1a-9d4a-32c6a397c74a.png)